### PR TITLE
[retiarii] add doc: customizing a new exploration strategy

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

**Review by ultmaster — APPROVED** [review]

File: docs/en_US/NAS/retiarii/Tutorial.rst [hunk 1]
```diff
@@ -23,26 +23,8 @@ Define Base Model
...
-        ...
-      def forward(self, x):
-        for i in range(10): # <- adhoc loop
-          ...
-
-* The candidate ops in ``LayerChoice`` should be decorated as blackbox module. For example, ``self.op = nn.LayerChoice([Op1(...), Op2(...), Op3(...)])``, where ``Op1``, ``Op2``, ``Op3`` should be decorated.
-* When users want to use ``ValueChoice`` in a module's input argument, the module should be decorated as blackbox module. For example, ``self.conv = MyConv(kernel_size=nn.ValueChoice([1, 3, 5]))``, where ``MyConv`` should be decorated.
-* If no mutation is targeted on a module, this module *can be* decorated as a blackbox module.
+* Replace the code ``import torch.nn as nn`` with ``import nni.retiarii.nn.pytorch as nn`` for PyTorch modules, such as ``nn.Conv2d``, ``nn.ReLU``.
+* Some **user-defined** modules should be decorated with ``@blackbox_module``. For example, user-defined module used in ``LayerChoice`` should be decorated.Users can refer to `here <#blackbox-module>`__ for detailed usage instruction of ``@blackbox_module``.
```
Comment (ultmaster) [inline docs/en_US/NAS/retiarii/Tutorial.rst:27]: should be decorated. <space> Users

File: nni/retiarii/experiment.py [hunk 2]
```diff
@@ -172,11 +172,16 @@ def run(self, config: RetiariiExeConfig = None, port: int = 8080, debug: bool =
...
-    def export_top_models(self, top_n: int):
+    def export_top_models(self, top_n: int = 1):
         """
         export several top performing models
         """
-        raise NotImplementedError
+        if top_n != 1:
+            _logger.warning('Only support top_n is 1 for now.')
+        if isinstance(self.trainer, BaseOneShotTrainer):
+            self.trainer.export()
```
Comment (ultmaster) [inline nni/retiarii/experiment.py:182]: return

File: docs/en_US/NAS/retiarii/Tutorial.rst [hunk 3]
```diff
@@ -71,13 +53,13 @@ Users can refer to :githublink:`Darts base model <test/retiarii_test/darts/darts
...
+A base model is only one concrete model not a model space. We provide APIs and primitives for users to express how the base model can be mutated, i.e., a model space which includes many models.
 
 **Express mutations in an inlined manner**
 
-For easy usability and also backward compatibility, we provide some APIs for users to easily express possible mutations during defining a base model. The APIs can be used just like PyTorch module.
+For easy usability and also backward compatibility, we provide some APIs for users to easily express possible mutations after defining a base model. The APIs can be used just like PyTorch module.
 
-* ``nn.LayerChoice``. It allows users to put several candidate operations (e.g., PyTorch modules), one is chosen in each explored model. *Note that the candidates should be decorated as blackbox module.*
+* ``nn.LayerChoice``. It allows users to put several candidate operations (e.g., PyTorch modules), one of them is chosen in each explored model. *Note that the candidate is a user-defined module, it should be decorated as blackbox module.*
```
Comment (colorjam) [inline docs/en_US/NAS/retiarii/Tutorial.rst:63]: do the candidates denote `ops.PoolBN` and `ops.SepConv`?

File: docs/en_US/NAS/retiarii/Tutorial.rst [hunk 4]
```diff
@@ -71,13 +53,13 @@ Users can refer to :githublink:`Darts base model <test/retiarii_test/darts/darts
...
```
Comment (colorjam) [inline docs/en_US/NAS/retiarii/Tutorial.rst:62]: does the candidate denote ops.PoolBN and ops.SepConv?

**Review by colorjam — APPROVED** [review]

File: docs/en_US/NAS/retiarii/Tutorial.rst (hunk 4 above)
Comment (QuanluZhang) [reply docs/en_US/NAS/retiarii/Tutorial.rst:62]: in the following example, yes. I will make it clear

**Review by liuzhe-lz — APPROVED** [review]

**Pull request merged**